specification/maps/data-plane/Search/preview/1.0/search.json

```diff
@@ -143,15 +150,120 @@
       "in": "query",
       "description": "The applicable query specified as a comma separated string composed by latitude followed by longitude e.g. \"47.641268,-122.125679\".",
       "required": true,
-      "type": "string",
-      "x-ms-parameter-location": "method"
+      "x-ms-parameter-location": "method",
+      "type": "array",
+      "collectionFormat": "csv",
+      "minItems": 2,
+      "maxItems": 2,
+      "items": {
+        "type": "number",
+        "format": "double"
+      }
     },
     "View": {
       "name": "view",
       "in": "query",
```

Comment:
This should align with the name used in https://github.com/Azure/azure-rest-api-specs/pull/16162. In reality, it seems like this param should be entirely removed from this Swagger and instead referenced from `common.json`

Reply:
I've modified the #16162 to use the exact definition of `View` described here. And the references to `common.json` are also made there.